Source/WebCore/dom/ElementRareData.h

```diff
@@ -199,7 +204,9 @@ class ElementRareData : public NodeRareData {
     std::unique_ptr<IntersectionObserverData> m_intersectionObserverData;
 
     std::unique_ptr<ResizeObserverData> m_resizeObserverData;
-    RefPtr<ResizeObserverSize> m_lastRememberedSize;
+
+    LayoutUnit m_lastRememberedLogicalWidth { -1 };
+    LayoutUnit m_lastRememberedLogicalHeight { -1 };
```

Comment:
I can't find any match for `LayoutUnit m_.*-1`, but there are some `optional<LayoutUnit> m_` (and most seem non-negative). So it may be better to continue the pattern? Though I guess it would use more memory, so not sure.

Reply:
Yes, that's true. Maybe `std::optional<LayoutUnit>` is better, to keep code consistent.